streams/src/main/java/org/apache/kafka/streams/kstream/internals/SessionWindowedCogroupedKStreamImpl.java

```diff
@@ -106,7 +108,7 @@ public KTable<Windowed<K>, V> aggregate(final Initializer<V> initializer,
             sessionMerger);
     }
 
-    private  StoreBuilder<SessionStore<K, V>> materialize(final MaterializedInternal<K, V, SessionStore<Bytes, byte[]>> materialized) {
+    private StoreBuilder<SessionStore<K, V>> materialize(final MaterializedInternal<K, V, SessionStore<Bytes, byte[]>> materialized) {
         SessionBytesStoreSupplier supplier = (SessionBytesStoreSupplier) materialized.storeSupplier();
         if (supplier == null) {
```

Comment:
ditto here -- seems like we can simplify things to make the `supplier` variable final.

I'm going to stop commenting on this because I suspect the pattern repeats a lot, so I'll leave it up to you to make a pass over the rest. And I know it feels petty, but given the logic around initializing and properly configuring the default store suppliers is relatively complex, I actually do believe there's value in trying to make things as straightforward as possible, including using `final` variables for things like this.